rest/doc_api.go

```diff
@@ -356,6 +356,62 @@ func (h *handler) handlePutAttachment() error {
 	return nil
 }
 
+func (h *handler) handleDeleteAttachment() error {
+	if h.isReadOnlyGuest() {
+		return base.HTTPErrorf(http.StatusForbidden, auth.GuestUserReadOnly)
+	}
+
+	docid := h.PathVar("docid")
+	attachmentName := h.PathVar("attach")
+	attachmentContentType := h.rq.Header.Get("Content-Type")
```

Comment:
Content type is not used or needed for a delete so this can be removed, as well as from the docs

Reply:
Yeah good point, I will get this updated in the morning.